platform/viewer/cypress/support/commands.js

```diff
@@ -30,12 +30,73 @@ Cypress.Commands.add('openStudy', (patientName) => {
     cy.visit('/');
     cy.get('#patientName')
       .type(patientName);
-    cy.get('.studylistStudy > .patientName')
-      .contains(patientName)
-      .click();
+
+    const patientResult = '.studylistStudy > .patientName';
+
+    cy.get(patientResult)
+      .then({ timeout: 5000 }, ($patientResult) => {
+        cy.contains(patientName)
+        .click();
+      })
   }
 );
 
 Cypress.Commands.add('drag', {prevSubject: 'element',},
   (...args) => DragSimulator.simulate(...args)
 );
+
+// Command to perform two clicks into two different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addLine', (element, initPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
+    .then(() =>{
+      cy.get(element)
+        .trigger('mousemove', { clientX: finalPosition[0], clientY: finalPosition[1] })
+        .click({ force: true })
+    })
+});
+
+
+// Command to perform three clicks into three different positions. Each position must be (x, y).
+// The positions are considering the element as reference, therefore, top-left of the element will be (0, 0).
+Cypress.Commands.add('addAngle', (element, initPosition, midPosition, finalPosition) =>  {
+
+  cy.get(element)
+    .click(initPosition[0], initPosition[1], { force: true })
```

Comment:
Does our element fail an [actionability check](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability)? What's causing us to use `force` here? Those checks are supposed to be in place to make sure we can only click things the user can actually click.

Reply:
Yeah, unfortunately, it is necessary not only to `force` the click but also to `trigger` a mouse movement to the next position. Otherwise, the annotation was not being added.